less/mixins/sequence.less

```diff
@@ -140,3 +91,53 @@
     margin-left: (-@sequence-item-padding - (@sequence-icon-lg / 2) + 1px);
   }
 }
+
+
+.sequence-variant(
```

Comment:
Would a comment here for future posterity add value? For example why we use so many gradients?